app/utilities/accessibility.js

```diff
@@ -41,4 +42,18 @@ export const getWCAG2TextSize = el => {
   )
 
   return  isLarge ? 'Large' : 'Small'
+}
+
+export const getContrastingColor = color => {
+  const randomColor = random().toHexString()
+  const oppositeHue = new TinyColor(color).spin(180)
+  return mostReadable(
+    color,
+    ['hotpink', randomColor, oppositeHue],
+    {
+      includeFallbackColors: true,
+      level: 'AAA',
+      size: 'small'
+    }
+  )
```

Comment:
I forgot to mention [I wrote one here](https://github.com/GoogleChromeLabs/ProjectVisBug/blob/main/app/features/color.js#L176), for the icons overlaying the colors in the sidebar, but I like the use of `mostReadable()` here. When i tried this out locally, sometimes pretty odd colors would be chosen, like the random or the inversion/180 rotate, which produced some pretty bad combos. I updated it to this, which felt more stable:

```suggestion
export const getContrastingColor = color => {
  return mostReadable(
    color,
    ['#000', '#fff'],
    {
      includeFallbackColors: true,
      level: 'AAA',
      size: 'small'
    }
  )
```

but i also didnt hate using the existing one with a tweak for more contrast. Sometimes it picks a color in the same family as what it's displaying, which looks really nice. and it clips to black/white:

```js
  let contrast = new TinyColor(color)

  contrast = contrast.isDark()
    ? contrast.lighten(75)
    : contrast.darken(75)

  return contrast
```

fun! i don't know what's best yet 🙂

Reply:
btw i read that when you have `includeFallbackColors: true` set, then `mostReadable` will automatically fall back to black or white if none of the items in the array param have good enough contrast, so i think i actually prefer your existing function

Reply:
actually, I think I'll use your suggestion above. i tried the tweak on the existing one and sometimes got low contrast (i'm in dark mode). 
<img width="430" alt="Screen Shot 2021-05-12 at 20 10 58" src="https://user-images.githubusercontent.com/18131787/118059423-7ff51d00-b35e-11eb-8d0e-198882c697ee.png">